package.json

```
@@ -16,7 +16,7 @@
     "@patternfly/react-tokens": "^4.58.2",
     "@reduxjs/toolkit": "~1.6.x",
     "@sentry/browser": "5.19.1",
-    "axios": "^0.21.2",
+    "axios": "^0.22.0",
```

```suggestion
    "axios": ">=0.22.0 <1.0.0",
```

Usage of caret for packages starting with a 0-major version is problematic. It only updates the patch version!

---

Ok!